config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml

```diff
@@ -28,17 +30,20 @@ patient:
     - high
     - abnormal
     type: string
-    search_term: elevated alkaline phosphatase  

```

Comment:
@karafecho I see ```1```value for ```name_lookup``` in the updated covid feature yaml file, so looks like you are using the old yaml file that did not pass yaml validator.